package.json

```diff
@@ -9,11 +9,13 @@
     "postversion": "git push && git push --tags && npm run clean && npm run docs:publish",
     "version": "npm run build",
     "clean": "rimraf build",
-    "lint": "eslint src/**",
-    "test": "mocha --compilers js:babel/register --recursive src/**/__tests__/*.js",
+    "lint": "eslint src/** packages/**/src/**",
     "check": "npm run lint && npm run test:all",
-    "build": "babel src --out-dir build",
-    "test:watch": "mocha --compilers js:babel/register --recursive src/**/__tests__/*.js --watch",
+    "build": "npm run build:core && npm run build:packages",
+    "build:core": "babel src --out-dir build",
+    "build:packages": "bash ./scripts/build-packages.sh",
+    "test": "mocha --compilers js:babel/register test/*.js packages/**/test/*.js",
```

Comment:
since packages have their own tests, this should probably run `npm test` in that directory, rather than baking subpackage-specific paths into the root package.json.

in other words, the packages should be as separable as possible, such that the decision to put them in a separate repo or not can be made and changed without impacting much.